airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -88,6 +88,15 @@
       }
     }
   },
+  "api_url": {
+    "type": "string",
+    "examples": ["api.github.com"],
+    "pattern": "^(?!http(s)?:)[a-z0-9-]+:\/\/.+$|^\/.*$",
+    "title": "API URL",
+    "default": "api.github.com",
+    "description": "Please enter your self-hosted URL from Github.",
+    "order": 1
```

Comment:
This is probably "order: 99" as we want this at the end of the page (it's optional and most users don't need it)